data/map.txt

```diff
@@ -29279,6 +29316,12 @@ planet "Pelubta Station"
 	spaceport `The intake section has a consistently high traffic, mostly made up of Coalition merchants supplying the station and leaving with shipments of fuel to take to other Arach worlds. The science section seems to be completely closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes.`
 	"required reputation" 20
 
+planet "Periaxle Circuit"
+	landscape
+	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the three species fought the Quarg. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the War of Independence, and the whole structure transformed into a gigantic particle accelerator, used by the Saryds to research weapons and shielding technology. Nowadays, the machinery remains working as it did before, but its purpose has shifted to a more general purpose research facility.`
+	description `	First completed dozens of centuries before the war, the structure's original platings failed to last the test of millennia unscathed like the Quarg's own, and are in a constant need of replacement to prevent the miniature ring's integrity from being compromised. Every so often, a crew of maintenance craft can be seen conducting repairs on some outer section of the ring.`
```

Comment:
```suggestion
	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the war, and the station transformed into a gigantic particle accelerator, used by the Saryds to research weapons and shielding technology. Nowadays, its purpose has shifted into a more general purpose research facility.`
	description `	Completed dozens of centuries before the war, the structure's original platings failed to last the test of millennia unscathed like the Quarg's own, and are in a constant need of replacement to prevent the miniature ring's integrity from being compromised. Every so often, maintenance craft can be seen conducting repairs on some outer section of the ring.`
```